docs/howto/garbage-collection/standalone-gc.md

```diff
@@ -149,41 +170,75 @@ docker run \
+
+First, change the permissions for `~/.aws/*` to allow the docker container to read this directory:
+```bash
+chmod 644 ~/.aws/*
+```
+
+Then, run the docker image and mount `~/.aws` to the `lakefs-sgc` home directory on the docker container:
+```bash
+docker run \
+--network=host \
+-v ~/.aws:/home/lakefs-sgc/.aws \
+-e AWS_REGION=us-east-1 \
+-e AWS_PROFILE=<your profile> \
+-e LAKEFS_SGC_LAKEFS_ENDPOINT_URL=<your endpoint URL> \
+-e LAKEFS_SGC_LAKEFS_ACCESS_KEY_ID=<your lakefs accesss key> \
+-e LAKEFS_SGC_LAKEFS_SECRET_ACCESS_KEY=<your lakefs secret key> \
+-e LAKEFS_SGC_LOGGING_LEVEL=debug \
+treeverse/lakefs-sgc:<tag> run <repository>
+```
 ### Output
 `lakefs-sgc` will write its reports to `<REPOSITORY_STORAGE_NAMESPACE>/_lakefs/retention/gc/reports/<RUN_ID>/`. \
 _RUN_ID_ is generated during runtime by the Standalone GC. You can find it in the logs:
 ```
 "Marking objects for deletion" ... run_id=gcoca17haabs73f2gtq0
 ```
 
-In this prefix, you'll find 3 objects:
-- `deleted.json` - Containing all marked objects in a json format. 
-- `deleted.parquet` - Containing all marked objects in a parquet format, with the following schema:
+In this prefix, you'll find 2 objects:
+- `deleted.csv` - Containing all marked objects in a CSV containing one `address` column. Example:
+   ```csv
+   address
+   "data/gcnobu7n2efc74lfa5ug/csfnri7n2efc74lfa69g,_e7P9j-1ahTXtofw7tWwJUIhTfL0rEs_dvBrClzc_QE"
+   "data/gcnobu7n2efc74lfa5ug/csfnri7n2efc74lfa78g,mKZnS-5YbLzmK0pKsGGimdxxBlt8QZzCyw1QeQrFvFE"
+   ...
    ```
-  ┌─────────────┬─────────────┬─────────┬─────────┬─────────┬─────────┐
-  │ column_name │ column_type │  null   │   key   │ default │  extra  │
-  │   varchar   │   varchar   │ varchar │ varchar │ varchar │ varchar │
-  ├─────────────┼─────────────┼─────────┼─────────┼─────────┼─────────┤
-  │ address     │ VARCHAR     │ YES     │         │         │         │
-  └─────────────┴─────────────┴─────────┴─────────┴─────────┴─────────┘
-  ```
 - `summary.json` - A small json summarizing the GC run. Example:
-```json
-{
-    "run_id": "gcoca17haabs73f2gtq0",
-    "success": true,
-    "first_slice": "gcss5tpsrurs73cqi6e0",
-    "start_time": "2024-10-27T13:19:26.890099059Z",
-    "cutoff_time": "2024-10-27T07:19:26.890099059Z",
-    "num_deleted_objects": 33000
-}
-```
+   ```json
+   {
+       "run_id": "gcoca17haabs73f2gtq0",
+       "success": true,
+       "first_slice": "gcss5tpsrurs73cqi6e0",
+       "start_time": "2024-10-27T13:19:26.890099059Z",
+       "cutoff_time": "2024-10-27T07:19:26.890099059Z",
+       "num_deleted_objects": 33000
+   }
+   ```
 
 ### Deleting marked objects
-To delete the objects marked by the GC, you'll need to read the `deleted.parquet` or `deleted.json` files, and manually delete each address from AWS.
+
+To delete the objects marked by the GC, you'll need to read the `deleted.csv` file, and manually delete each address from AWS.
+
+Example bash command to move all the marked objects to a different bucket on S3:
```

Comment:
I would add a note about playing it safe and moving the objects instead of deleting them

Reply:
Done